test/integration/select_test.lua

```diff
@@ -133,12 +138,35 @@ add('test_select_all', function(g)
 
     table.sort(customers, function(obj1, obj2) return obj1.id < obj2.id end)
 
+    local result, err = g.cluster.main_server.net_box:eval([[
+        local crud = require('crud')
+
+        local objects, err = crud.select('customers', nil)
+        return objects, err
+    ]])
+
+    t.assert_equals(err, nil)
+    t.assert_equals(result.rows, {
+        {1, 477, "Elizabeth", "Jackson", 12, "New York"},
+        {2, 401, "Mary", "Brown", 46, "Los Angeles"},
+        {3, 2804, "David", "Smith", 33, "Los Angeles"},
+        {4, 1161, "William", "White", 81, "Chicago"},
+    })
+    t.assert_equals(result.metadata, {
+        {name = 'id', type = 'unsigned'},
+        {name = 'bucket_id', type = 'unsigned'},
+        {name = 'name', type = 'string'},
+        {name = 'last_name', type = 'string'},
+        {name = 'age', type = 'number'},
+        {name = 'city', type = 'string'},
+    })
+
     -- no after
     local objects, err = g.cluster.main_server.net_box:eval([[
         local crud = require('crud')
 
         local objects, err = crud.select('customers', nil)
```

Comment:
Is it objects? Maybe is should be called `res` or `result`?

Reply:
Well, fixed